packit/patches.py

```diff
@@ -231,6 +235,76 @@ def linearize_history(git_ref: str):
             env={"FILTER_BRANCH_SQUELCH_WARNING": "1"},
         )
 
+    @staticmethod
+    def process_patches(
+        patches: Dict[str, str], commits: List[git.Commit], patch_list: List
+    ):
```

Comment:
Looking at the code I don't see a reason why it should modify an existing list

```suggestion
        patches: Dict[str, str], commits: List[git.Commit]) -> List[PatchMetadata]:
        patch_list: List[PatchMetadata] = []
        ...
        return patch_list
```